### PR TITLE
Add modal windows 

### PR DESCRIPTION
**Pull request by Ruthmy**
In this milestone:
We implement a popup window that includes the project details in both desktop and mobile.

- [x] Created the `cards.js` file to store all code about the modal windows functionality.
- [x] Created a JavaScript array to store all the project's information.
- [x] The JavaScript object, contain the minimum requirements: name, description, featured image, technologies, link to live version, link to source.
- [x] Update de index.html file, now the `Recent Works` section is created dynamically using the information stored in that JavaScript array.
- [x] Added the desktop modal version and the CSS style for it.
- [x] Added and working the See Project button on the cards to show up the modal window.
- [x] Added and working the Close button on the cards to close de modal.
- [x] Added the mobile modal version and the CSS style for it.

Thank you very much for review my code.
Don't hesitate to ask for changes if needed.

### CODE REVIEW

File: js/cards.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                        <li class="recent-works__tag">${singleCardContents.languages[2]}</li>
+                        <li class="recent-works__tag">${singleCardContents.languages[3]}</li>
+                    </ul>
+                    <button class="recent-works__link" href="#">See Project</button>
+                </div>
+            </div>
+        </section>
+        `;
+  // uncomment the line below to see the output in the browser console.
+  // console.log(insideCard);
```
Comment (Shedrack-Sunday) [inline js/cards.js]: - [ ] As this is simulating a review for a production-level code, kindly remove the `console` comments, as your reviewer is at the discretion to use a professional way to debug your code, which means your responsibility is to keep your code as clean as possible. such information can be added to your README under `usage` if you want to 👍

File: js/menujs.js [hunk 2]
```diff
@@ -9,7 +10,7 @@ hamburger.addEventListener('click', (event) => {
   hamburger.classList.toggle('showMobil');
 });
 
-console.log(arrAncherTag);
+// console.log(arrAncherTag);
 
 arrAncherTag.forEach((a) => {
```
Comment (Shedrack-Sunday) [inline js/menujs.js]: - [ ] Please make reference to the line `101` to see my detailed review on wy you should remove the `console.log` comment 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Ruthmy     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [ ] I must admit you have really put in the effort so far in this project, however, the changes required have to do with a bit of styling.

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @Ruthmy 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**